.github/workflows/BuildPR.yml

```diff
@@ -27,12 +27,14 @@ jobs:
       - name: pre build
         run: cd External/objective-git && script/bootstrap && script/update_libgit2 && cd ../..
       - name: Build project
+        run: xcodebuild -workspace GitX.xcworkspace -scheme GitX -archivePath ./GitX archive
```

Comment:
Does it maybe help when you add  `archive ARCHS="x86_64 arm64"` ?

https://github.com/gitx/objective-git/blob/master/.github/workflows/BuildPR.yml#L22

Reply:
I just pushed this. If it builds, let me know if it helps.

Reply:
It also feels curious that #234 (the M1 support PR) also has a lot of openssl related issues.

Reply:
It finished, but I don't think it did anything:
```
Command line invocation:
    /Applications/Xcode_13.2.1.app/Contents/Developer/usr/bin/xcodebuild -workspace GitX.xcworkspace -scheme GitX -archivePath ./GitX archive "ARCHS=x86_64 arm64"

User defaults from command line:
    IDEArchivePathOverride = /Users/runner/work/gitx/gitx/GitX
    IDEPackageSupportUseBuiltinSCM = YES

Build settings from command line:
    ARCHS = x86_64 arm64

--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:x86_64, id:4203018E-580F-C1B5-9525-B745CECA79EB }
```

I don't see any other reference to arm64. 😕